### PR TITLE
chore: different POIs improvements

### LINKED ISSUE #1901: Improvements on POIs inside the map
- [ ] POI star must be printed exactly on top of the coordinates, and not displaced depending of the text
This is wrong

This is (almost) right

- [ ] POIs text needs to be limited to a maximum of 2 lines. Rest of characters needs to be trimmed

- [ ] POIs with "Empty parcel" and no data must be hidden from the map

### PR DESCRIPTION
**Pull request by davidejensen**
## What does this PR change?

Fix #1901
Removes default name POIs, reduces number of characters of title to avoid more than 2 lines of text, centers the POI icon in the square of the parcel

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/poi-improvements
2. Verify in the map the changes listed above

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Comment by davidejensen** [comment]
Regarding the point "POI star must be printed exactly on top of the coordinates, and not displaced depending of the text" we need to discuss it @olavra 
I've applied those changes but there are some problems with it, before the star was placed in the geometrical center of the group of parcels with the same name, now I calculate the most centered parcel of the group and center it there, i don't know if that's the expected behavior

**Comment by davidejensen** [comment]
Can be reviewed, changes have been approved

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs [hunk 1]
```diff
@@ -266,6 +268,9 @@ private void MapRenderer_OnSceneInfoUpdated(MinimapMetadata.MinimapSceneInfo sce
             if (scenesOfInterest.Contains(sceneInfo))
                 return;
 
+            if (sceneInfo.name != null && sceneInfo.name.Equals(EMPTY_PARCEL_NAME))
```
Comment (lorux0) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs:271]: could you extract into a method: `IsAnEmptyParcel`

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs [hunk 2]
```diff
@@ -278,13 +283,24 @@ private void MapRenderer_OnSceneInfoUpdated(MinimapMetadata.MinimapSceneInfo sce
             }
 
             centerTile /= (float)sceneInfo.parcels.Count;
+            float distance = float.PositiveInfinity;
+            Vector2 centerParcel = Vector2.zero;
+            foreach (var parcel in sceneInfo.parcels)
```
Comment (lorux0) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs:288]: you can extract this block into a method in `MinimapMetadata`: `GetNearestParcel(Vector2 from)`

**Review by lorux0 — CHANGES_REQUESTED** [review]

**Review by lorux0 — APPROVED** [review]

**Pull request merged**